website/assets/scss/_breadcrumb.scss

```diff
@@ -0,0 +1,45 @@
+.breadcrumb {
+    display: flex;
+    flex-wrap: wrap;
+    //padding: $breadcrumb-padding-y;
```

Comment:
Can these commented CSS directives be removed

Reply:
Done